sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -788,8 +788,18 @@ inlineTable
     : VALUES expression (COMMA expression)* tableAlias
     ;
 
+functionTableRelationArgument
```

Comment:
I would name these `functionTableSubqueryArgument` and `functionTableReferenceArgument`, respectively, to help differentiate between the two (since both are relations).